frontend/src/components/question-creation/question-elements/modals/multiple-choice/MultipleChoiceModal.tsx

```diff
@@ -33,7 +33,7 @@ const MultipleChoiceModal = ({
   const [emptyOptionError, setEmptyOptionError] = useState(false);
 
   useEffect(() => {
-    setOptionCount(data ? data.optionCount : 0);
+    setOptionCount(data ? data.options.length : 0);
```

Comment:
i think it still makes sense to keep `optionCount`? bc we use it as a local variable to add elements to `options` (so if we just used `options.length` it would always be 0)

Reply:
ah kk